README.md

```diff
@@ -84,10 +84,6 @@ This is convenient for development for a number of reasons:
 
 There are however a number of downsides:
 
-* Only one mainstream MPI distribution ([MPICH](https://www.mpich.org/)) supports
-  nested calls to `MPI_Init`. If your 'parent' `pytest` process initialises MPI
-  (for instance by executing `from mpi4py import MPI`) then this will cause non-MPICH
-  MPI distributions to crash.
```

Comment:
Has this actually changed? What about old versions?

Reply:
I don't know about older versions, but it magically seems to work now: https://github.com/firedrakeproject/mpi-pytest/pull/21

Reply:
Yes this changed a bit ago, but we're not entirely sure why it now works with OpenMPI.
#21 added OpenMPI forking mode, and adds tests for it to the CI.